inst/tinytest/test-cl-request.R

```diff
@@ -131,7 +131,9 @@ params = list(
   "param2" = "value2"
 )
 b = make_multipart_body(params, files)
```

Comment:
Since `make_multipart_body` is randomized, let's add `set.seed()` for reproducibility.

Reply:
Already done with [`tests/tinytest.R`](tests/tinytest.R#L3)